otx/recipes/stages/segmentation/incremental.py

```diff
@@ -1,29 +1,29 @@
-_base_ = ["./train.py", "../_base_/models/segmentors/seg_class_incr.py", "../_base_/data/custom_seg.py"]
+_base_ = ["./train.py", "../_base_/models/segmentors/segmentor.py", "../_base_/data/custom_seg.py"]
 
-optimizer = dict(_delete_=True, type="Adam", lr=1e-3, eps=1e-08, weight_decay=0.0)
-
-optimizer_config = dict(
+optimizer = dict(
     _delete_=True,
-    grad_clip=dict(
-        # method='adaptive',
-        # clip=0.2,
-        # method='default',
-        max_norm=40,
-        norm_type=2,
-    ),
+    type="AdamW",
+    lr=0.0005,
+    betas=(0.9, 0.999),
+    weight_decay=1e-4,
+    paramwise_cfg={"bias_decay_mult ": 0.0, "norm_decay_mult ": 0.0},
```

Comment:
Could you explain what is this?

Reply:
it is No Bias Decay, it will remove decay for norm layers and bias